src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NzbDrone.Core.Annotations;
+
+namespace NzbDrone.Core.Indexers.Newznab
+{
+    public static class NewznabCategoryFieldOptionsConverter
+    {
+        public static List<FieldSelectOption> GetFieldSelectOptions(List<NewznabCategory> categories)
+        {
+            // Ignore categories not relevant for Readarr
+            var ignoreCategories = new[] { 1000, 2000, 3000, 4000, 5000, 6000 };
+
+            // And maybe relevant for specific users
+            var unimportantCategories = new[] { 0 };
+
+            var result = new List<FieldSelectOption>();
+
+            if (categories == null)
+            {
+                // Fetching categories failed, use default Newznab categories
+                categories = new List<NewznabCategory>();
+                categories.Add(new NewznabCategory
+                {
+                    Id = 7000,
+                    Name = "Books",
+                    Subcategories = new List<NewznabCategory>
+                    {
+                        new NewznabCategory { Id = 7020, Name = "Ebook" },
+                        new NewznabCategory { Id = 7030, Name = "Comics" },
+                        new NewznabCategory { Id = 7040, Name = "Magazines" }
+                    }
+                });
+                categories.Add(new NewznabCategory
+                {
+                    Id = 8000,
```

Comment:
8000 should likely be in unimportantCategories, meaning we want to allow it, but not as default.

Reply:
Done